package.json

```diff
@@ -13,10 +13,7 @@
     "axios": "^1.7.2",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-<<<<<<< HEAD
```

Comment:
Conflict got pushed here

Reply:
just a package. Can you add the package react-lottie?